### ISSUE TITLE
rule 9591220 triggers false positive in wordpress wp-admin/site-health.php

### ISSUE THREAD

**Issue by Fregf** [coreruleset/coreruleset#1862]
When going to a Wordpress 5.4 installation /wp-admin/site-health.php (with CRS Wordpress exclusions activated), a 403 is returned with these errors in Apache log:
```
[Sun Aug 23 18:06:35.248782 2020] [:error] [pid 29617:tid 139745954658048] [client 192.168.0.10:50026] [client 192.168.0.10] ModSecurity: Warning. Pattern match "(?i)(?:System\\\\.Data\\\\.OleDb\\\\.OleDbException|\\\\[Microsoft\\\\]\\\\[ODBC SQL Server Driver\\\\]|\\\\[Macromedia\\\\]\\\\[SQLServer JDBC Driver\\\\]|\\\\[SqlException|System\\\\.Data\\\\.SqlClient\\\\.SqlException|Unclosed quotation mark after the character string|'80040e14' ..." at RESPONSE_BODY. [file "/usr/share/modsecurity-crs/rules/RESPONSE-951-DATA-LEAKAGES-SQL.conf"] [line "340"] [id "951220"] [msg "mssql SQL Information Leakage"] [data "Matched Data: SQL server is up to date\\x22,\\x22status\\x22:\\x22good\\x22,\\x22badge\\x22:{\\x22label\\x22:\\x22Performance\\x22,\\x22color\\x22:\\x22blue\\x22},\\x22description\\x22:\\x22<p>The SQL server is a required piece of software for the database WordPress uses to store all your site&#8217;s content and settings.<\\x5c/p>\\x22,\\x22actions\\x22:\\x22<p><a href=\\x5c\\x22https:\\x5c/\\x5c/wordpress.org\\x5c/about\\x5c/requirements\\x5c/\\x5c\\x22 target=\\x5c\\x22_blank\\x5c\\x22 rel=\\x5c\\x22noopener noreferrer\\x5c\\x22>Learn more ..."] [severity "CRITICAL"] [ver "OWASP_CRS/3.3.0"] [tag "applic [hostname "example.org"] [uri "/wp-admin/site-health.php"] [unique_id "X0KUCi3UFxmAiUfOLZfC0AAAAc8"], referer: https://example.org/wp-admin/
[Sun Aug 23 18:06:35.266670 2020] [:error] [pid 29617:tid 139745954658048] [client 192.168.0.10:50026] [client 192.168.0.10] ModSecurity: Access denied with code 403 (phase 4). Operator GE matched 4 at TX:outbound_anomaly_score. [file "/usr/share/modsecurity-crs/rules/RESPONSE-959-BLOCKING-EVALUATION.conf"] [line "76"] [id "959100"] [msg "Outbound Anomaly Score Exceeded (Total Score: 5)"] [ver "OWASP_CRS/3.3.0"] [tag "anomaly-evaluation"] [hostname "example.org"] [uri "/wp-admin/site-health.php"] [unique_id "X0KUCi3UFxmAiUfOLZfC0AAAAc8"], referer: https://example.org/wp-admin/
[Sun Aug 23 18:06:35.267263 2020] [:error] [pid 29617:tid 139745954658048] [client 192.168.0.10:50026] [client 192.168.0.10] ModSecurity: Warning. Operator GE matched 4 at TX:outbound_anomaly_score. [file "/usr/share/modsecurity-crs/rules/RESPONSE-980-CORRELATION.conf"] [line "102"] [id "980140"] [msg "Outbound Anomaly Score Exceeded (score 5): individual paranoia level scores: 5, 0, 0, 0"] [ver "OWASP_CRS/3.3.0"] [tag "event-correlation"] [hostname "example.org"] [uri "/wp-admin/site-health.php"] [unique_id "X0KUCi3UFxmAiUfOLZfC0AAAAc8"], referer: https://example.org/wp-admin/
```

[*] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

This rule added to /usr/share/modsecurity-crs/rules/REQUEST-903.9002-WORDPRESS-EXCLUSION-RULES.conf fixes the issue:
```
SecRule REQUEST_FILENAME "@endsWith /wp-admin/site-health.php" \
    "id:9002901,\
    phase:2,\
    pass,\
    t:none,\
    nolog,\
    ctl:ruleRemoveById=951220,\
    ver:'OWASP_CRS/3.3.0'"
```
I can make a pull request if you think this is OK.

**Comment by lifeforms** [comment]
Hi @Fregf, thanks for your contribution! The exclusion rule that you proposed is looking great. A PR would be most welcome!

**Comment by lifeforms** [comment]
I've created the PR to fix this issue in #1895. It should become part of our next release. Thanks for your report!
